src/content/en/updates/2018/07/page-lifecycle-api.md

```diff
@@ -0,0 +1,962 @@
+   non-capturing event listeners to a common ancestor element and observe all
+   of them.
+3. The capture phase executes before the target or bubble phases, so adding
+   listeners there helps ensures they run before other code can cancel them.
+
+### Managing cross-browsers differences {: #managing-cross-browsers-differences}
+
+The chart in the beginning of this article outlines the state and event flow
+according to the Page Lifecycle API. But since this API has just been
+introduced, the new events and DOM APIs have not been implemented in all
+browsers.
+
+Furthermore, the events that are implemented in all browsers today are not
+implemented consistently. For example:
+
+1. Some browsers do not fire a `blur` event when switching tabs. This means
+   (contrary to the diagram and tables above) a page could go from the active
+   state to the hidden state without going through passive first.
+2. Several browsers implement a page navigation cache (also known as a
+   back-forward cache), and the Page Lifecycle API classifies cached pages as
+   being in the frozen state. Since this API is brand new, these browsers do not
+   yet implement the `freeze` and `resume` events, though this state can still
+   be observed via the `pagehide` and `pageshow` events.
+3. Older versions of Internet Explorer (10 and below) do not implement the
+   `visibilitychange` event.
+4. The ordering of when `pagehide` and `visibilitychange` events are dispatched
+   for unloading pages has
+   [changed](https://github.com/w3c/page-visibility/issues/39). Previously
+   browsers would dispatch `visibilitychange` after `pagehide` if the page was
+   visible while being unloaded. New Chrome versions will dispatch `pagehide`
+   after `visibilitychange`, regardless of the document's visibility state prior
+   to the page being unloaded.
+
+To make it easier for developers to deal with these cross-browsers
+inconsistencies and focus solely on following [lifecycle best practices
+](#developer-recommendations-for-each-state), we've released
+[PageLifecycle.js](https://github.com/GoogleChromeLabs/page-lifecycle), a
+JavaScript library for observing Page Lifecycle API state changes.
+
+[PageLifecycle.js](https://github.com/GoogleChromeLabs/page-lifecycle)
+[normalizes cross-browser differences in event firing order so that state change
+[always occur exactly as outlined in the chart and tables in this article (and
+[do so consistently in all browsers).
+
+
+## Developer recommendations for each state {: #developer-recommendations-for-each-state}
+
+As developers, it's important to both understand Page Lifecycle states &mdash;
+as well as know how to observe them in code &mdash; because the type of work you
+should (and should not) be doing depends largely on what state your page is in.
+
+For example, it clearly doesn't make sense to be updating the inline styles of
+DOM elements if your page is in the hidden state. While this example is pretty
```

Comment:
Change to 'For example, it doesn't make sense to update the inline styles of DOM elements while the page's state is `hidden`.'

I didn't flag them all, but states and events should be have backticks around them. Please fix throughout.